### ISSUE TITLE
Form improvement for accessibility

### ISSUE THREAD

**Issue by jes237** [aalto-grades/aalto-grades#988]
As per the [design guide](https://github.com/aalto-grades/aalto-grades/wiki/Design-Guidelines) the required form fields should be indicated with text instead of only using colors or an asterisk (*). Currently only asterisks used and helper text that many times is small and does not explicitly say that it is required.

*The best way to indicate a required field is to include the word ‘required’ as part of the label. Many forms use a red asterisk, ‘*’, in the label. The asterisk is often not announced to screen reader users because it’s considered punctuation. People with low vision can often miss the ‘*’ too due to it’s small size and the color that is used.
...
A required field should also be flagged as required in the HTML code. This needs to include some visible indicator as well. If it doesn’t then everyone is likely to receive an error message that the required fields haven’t been completed.* ([source](https://www.w3.org/WAI/test-evaluate/easy-checks/required-fields/))
